topology/University of Chicago/UChicago/UChicago_OSGConnect.yaml

```
@@ -61,6 +61,162 @@ Resources:
     Tags:
       - OSPool
 
+  UChicago_OSGConnect_ap20:
+    Active: true
+    Description: The ap20.uc.osg-htc.org login node for the OSG Open Pool
+    ID: 
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Judith Stephen
+          ID: a418fbc5dd33637bba264c01d84d52dd317f2813
+        Secondary:
+          Name: David Jordan
+          ID: 3c60d9a835029389712c5f6cc30720bc5562519d
+        Tertiary:
+          Name: Farnaz Golnaraghi
+          ID: OSG1000221
+        Tertiary 2:
+          Name: Lincoln Bryant
+          ID: 0a22bab3de2d83d723811e3fb1ebca904e924a97
+
+      Security Contact:
+        Primary:
+          Name: Judith Stephen
+          ID: a418fbc5dd33637bba264c01d84d52dd317f2813
+        Secondary:
+          Name: David Jordan
+          ID: 3c60d9a835029389712c5f6cc30720bc5562519d
+        Tertiary:
+          Name: Farnaz Golnaraghi
+          ID: OSG1000221
+        Tertiary 2:
+          Name: Lincoln Bryant
+          ID: 0a22bab3de2d83d723811e3fb1ebca904e924a97
+    FQDN: ap20.uc.osg-htc.org
+    Services:
+      Submit Node:
+        Description: OS Pool access point
```

```suggestion
      Submit Node:
        Description: OS Pool access point
      XRootD origin server:
        Description: OSG Connect Origin Server
```

Since it's just another service running on the same host, we can specify it this way